metaflow/client/core.py

```diff
@@ -207,6 +207,15 @@ def default_namespace() -> str:
     return get_namespace()
 
 
+def inspect_spin(datastore_root):
```

Comment:
minor nit looking at the blog: we can make the default be "." since that will be the common case I suppose.